### ISSUE TITLE
IgnorePath rule not working properly in select cases

### ISSUE THREAD

**Issue by v1kn** [CyberShadow/aconfmgr#6]
IgnorePath '/boot' does not prevent `/boot/grub/grub.cfg` from being backed up. In the same way, '/usr/lib/modules' does not prevent kernel modules from being backed up.

    IgnorePath '/boot' 
    IgnorePath '/boot/*'

solves it, but Im not sure why only in those 2 cases the second line is necessary

**Comment by CyberShadow** [comment]
A glob is necessary to ignore modified files owned by packages.

Should the documentation be improved to emphasize this?

**Comment by v1kn** [comment]
Yes, as of now it is unclear why those 2 cases need two separate entries in ignore list.

**Comment by CyberShadow** [comment]
What is the origin of the first line?

**Comment by v1kn** [comment]
Not sure what you mean by that

**Comment by CyberShadow** [comment]
> Not sure what you mean by that

I meant to ask what inspired you to try the first line.

> Yes, as of now it is unclear why those 2 cases need two separate entries in ignore list.

https://github.com/CyberShadow/aconfmgr/compare/68ea74e2e8af9aa0ab2735295d21c05ad7f87d1b...e014657adb12fa20a917f392ce1bf1b7e58ba6ee

**Comment by v1kn** [comment]
I saw Your personal ignores file, and started to aggregate various specific ignore paths into larger general ones, like `/var/lib` or `/usr/lib`. I noticed that all I need is one high level parent entry to encompass all its children. 
Later, while browsing 99-unsorted file I noticed boot and modules entries that were supposed to be ignored. Tried globbing, which worked, and found the behavior inconsistent
